dts/bindings/misc/allegro,a4988.yaml

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) 2021 Teslabs Engineering S.L.
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
hey what's going on here, I've never written this binding, or did I?

Reply:
My bad forgot to adjust, fixed now.